### PR TITLE
Add duration maximums to tcld generation of certificates

### PR DESCRIPTION
**Pull request by rachfop**
## What does this PR do?

**Descirpiton**:
This pull request implements the changes mentioned in the Slack thread by Peter Sullivan on 7/24/2023, for the documentation file "certificates-issue.md." The changes include adding a paragraph, modifying a code block, and adjusting the formatting of the section headings and admonitions.

**Changes Made**:

Added a new paragraph between the first and second paragraphs to specify the maximum duration for a CA certificate and the appropriate duration for an end-entity certificate relative to its root CA certificate.
Modified the second code block, changing -d 1y to -d 364d to ensure proper certificate duration.
Removed the inline code formatting from "tcld" in the heading "Use tcld to generate certificates."
Moved the "Info" admonition about "max certs in a bundle" to appear immediately above the heading "Use tcld to generate certificates."

### CODE REVIEW

File: docs-src/cloud-context/certificates-issue.md [hunk 1]
```diff
@@ -18,10 +18,21 @@ When you configure the client SDK, you need to present a complete certificate ch
...
+The maximum payload size of a certificate bundle (before base64 encoding) is 32 KB.
+
+:::
+
+#### Use tcld to generate certificates
 
 CA and end-entity certificates can be generated with `tcld`.
 
+The maximum duration for a CA certificate is 1 year (`-d 1y`).
+
```
Comment (djmagee) [inline docs-src/cloud-context/certificates-issue.md:33]:
```suggestion
```

File: docs-src/cloud-context/certificates-issue.md [hunk 2]
```diff
@@ -18,10 +18,20 @@ When you configure the client SDK, you need to present a complete certificate ch
 
 If you don't have an existing certificate management infrastructure, issue the CA and end-entity certificates using `tcld` or open source tools such as certstrap.
 
-#### Use `tcld` to generate certificates
+:::info
+
+The maximum number of CA certificates in a certificate bundle is 16.
+The maximum payload size of a certificate bundle (before base64 encoding) is 32 KB.
```
Comment (flossypurse) [inline docs-src/cloud-context/certificates-issue.md:24]:
```suggestion
A certificate bundle can contain up to 16 CA certificates.
A certificate bundle can have a maximum payload size of 32 KB before base64 encoding.
```
quick switch to a more active voice - wdyt?

File: docs-src/cloud-context/certificates-issue.md [hunk 3]
```diff
@@ -18,10 +18,20 @@ When you configure the client SDK, you need to present a complete certificate ch
...
+
+:::
+
+#### Use tcld to generate certificates
 
 CA and end-entity certificates can be generated with `tcld`.
 
+The maximum duration for a CA certificate is 1 year (`-d 1y`).
+An end-entity certificate must expire before its root CA certificate, so specify its duration appropriately.
```
Comment (flossypurse) [inline docs-src/cloud-context/certificates-issue.md:33]:
```suggestion
A CA certificate can last up to 1 year (`-d 1y`).
You must set an end-entity certificate to expire before its root CA certificate, so specify its duration appropriately.
```

**Review by flossypurse — APPROVED** [review]
Great details.

**Pull request merged**